core/src/lib/alist.h

```diff
@@ -95,27 +101,6 @@ enum
  */
 
 template <typename T> class alist {
-  T* items = nullptr;
```

Comment:
yes true, just changed to follow the google standard https://google.github.io/styleguide/cppguide.html#Declaration_Order
in a way, it keeps the public side on top since that's what you are really looking for when using a class, it keeps access specifiers explicit without having to rely on implicit things.